crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -1387,6 +1619,29 @@ macro_rules! impl_tuple_fetch {
         /// SAFETY: each item in the tuple is read only
         unsafe impl<$($name: ReadOnlyWorldQuery),*> ReadOnlyWorldQuery for ($($name,)*) {}
 
+        #[allow(unused_variables)]
+        #[allow(non_snake_case)]
+        impl<'w, const N: usize, const ALIGN: usize, $($name: WorldQueryBatchGats<'w, N, ALIGN>),*> WorldQueryBatchGats<'w, N, ALIGN> for ($($name,)*) {
+            type FullBatch = ($($name::FullBatch,)*);
+        }
+
+        #[allow(unused_variables)]
+        #[allow(non_snake_case)]
+        #[allow(clippy::unused_unit)]
+        impl<const N: usize, const ALIGN: usize, $($name: WorldQueryBatch<N, ALIGN>),*> WorldQueryBatch<N, ALIGN> for ($($name,)*)
+        {
+            #[inline]
+            unsafe fn table_fetch_batched_aligned<'w>(
```

Comment:
SAFETY comment missing.

Reply:
Ditto here regarding `WorldQuery` and `table_fetch` (in case another reviewer comes across this)